docker-compose/readme.md

```diff
@@ -333,6 +393,9 @@ docker-compose up -d unittests
 
 ```
 
+## View Unit test logs
```

Comment:
I talked to Shivam about this offline, but this section appears to be mistakenly missing in the PR, he said he would add a commit to fix this.